app/assets/stylesheets/metadata.scss

```diff
@@ -201,4 +201,180 @@ dl.gem-c-metadata__list {
   padding: 15px 10px 15px 20px;
   margin-top: 60px;
   border-left: 5px solid #347ca9;
-}
\ No newline at end of file
+}
```

Comment:
If the entire contents of this file is responsible for styling the `citation` partial, let's call it that. Anything that is unrelated should be moved out.